docs/develop-guide.rst

```diff
@@ -336,38 +336,101 @@ Start the CLI to connect to the server and run SQL queries:
 7 How to run simple examples with Prestodb in distributed environment
 ---------------------------------------------------------------------
 
-(1) Create a folder to install cider files, for example ``cider``
-
-(2) | Copy the lib folder under the cider docker build environment to
-      every node, for example, copy
-      ``/usr/local/lib/`` folder to
-      ``/path/to/cider`` on every Prestodb node
+7.1 build presto in container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-   cp -a /usr/local/lib/ /path/to/cider
+   git clone --recursive https://github.com/Intel-bigdata/presto.git
+   pushd presto
+   git checkout -b BDTK origin/BDTK
+   pushd presto-native-execution
+   git clone --recursive https://github.com/intel/BDTK.git
+   pushd BDTK  
+   make release
+   popd
+   mkdir -p ./presto_cpp/main/lib
+   cp ./BDTK/build-Release/cider-velox/src/libvelox_plugin.a ./presto_cpp/main/lib
```

Comment:
We should provide a script to do deployment instead file-by-file copying so many so files. Your thoughts?

Reply:
fix, add build-presto-package.sh